lib/bamboo/email.ex

```diff
@@ -117,7 +117,12 @@ defmodule Bamboo.Email do
 
   for function_name <- @address_functions do
     @doc """
-    Sets the #{function_name} on the email
+    Sets the `#{function_name}` on the email
+
+    ```
+    new_email
+    |> #{function_name}(["sally@example.com", "james@example.com"])
```

Comment:
Also, what do you think of linking to the docs in Bamboo.Formatter? Something like

```
You can pass in a string, list of string, or anything that implements the `Bamboo.Formatter` protocol. See `Bamboo.Formatter` for more examples.
```

That might help people figure out what they can use for recipients. What do you think?